Cargo.toml

```diff
@@ -34,7 +34,7 @@ nb = "1"
 rand_core = "0.6"
 stm32f4 = "0.14.0"
 synopsys-usb-otg = { version = "0.2.0", features = ["cortex-m"], optional = true }
-sdio-host = { version = "0.5.0", optional = true }
+sdio-host = { version = "0.5.0", git = "https://github.com/antonok-edm/sdio-host", branch = "emmc", optional = true }
```

Comment:
What with this?

Reply:
https://github.com/jkristell/sdio-host/pull/10 is now merged upstream.

@jkristell - let me know when that makes it to a release on crates.io and I'll update the line here.

Reply:
sdio-host 0.6 should be on crates.io now!

Reply:
Awesome, updated here as well now.